### PR TITLE
fix: fix time stamp being too low

### PR DESCRIPTION
**Pull request by AlexLamDialpad**
# fix: fix time stamp being too low

## :hammer_and_wrench: Type Of Change

- [x] Fix
- [ ] Feature
- [ ] Refactoring
- [ ] Documentation

## :pencil: Checklist

- [x] I have reviewed my changes
- [ ] I have added tests
- [ ] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [ ] I have considered the performance impact of my change
- [ ] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

## :camera: Screenshots / GIFs

Before
image

After
image

## :link: Sources

https://dialpad.atlassian.net/browse/DP-80089

### CODE REVIEW

**Comment by AlexLamDialpad** [comment]
@braddialpad Yo I made adjusted this to line up to line-height 400 which is 24px.
The implementation in DP1 is using LH 300 which is 21px.
image

Based on the design here, the correct LH is 400 
https://www.figma.com/file/oTY56LgQha8ljb4c8w4GtO/DT-Shared%3A-DP-App-Components?type=design&node-id=9240-7140&mode=design&t=YIZq3PuHYrhRCi1u-0
image

So that's a lot of words to say that when you use this in DP1, gotta fix the LH because it's wrong

**Comment by iropolo** [comment]
Is there a reason why we are pushing `.lock` on this PR?

**Comment by AlexLamDialpad** [comment]
> Is there a reason why we are pushing `.lock` on this PR?

is someone else supposed to update this? Not sure as I didn't do this on purpose lol. Just ran npm install when I checked out main

**Comment by braddialpad** [comment]
> Is there a reason why we are pushing `.lock` on this PR?

Ah shit this was my bad. I think I forgot to push the lock on a prior update. Should be fine, just merge it.

**Comment by braddialpad** [comment]
> So that's a lot of words to say that when you use this in DP1, gotta fix the LH because it's wrong

Got it, thanks.

File: recipes/conversation_view/feed_item_row/feed_item_row.vue [hunk 1]
```diff
@@ -250,6 +250,16 @@ export default {
...
   transition-duration: 2s !important;
 
+  &__left-time {
+    color: var(--dt-color-foreground-tertiary)!important;
+    padding-top: var(--dt-space-350)!important;
+    line-height: var(--dt-font-line-height-400)!important;
+    font-size: var(--dt-font-size-100)!important;
+    font-weight: var(--dt-font-weight-normal)!important;
+    white-space: nowrap!important;
+    height: 100% !important;
```
Comment (braddialpad) [inline recipes/conversation_view/feed_item_row/feed_item_row.vue]: Don't think we need !important here do we?

Comment (AlexLamDialpad) [reply recipes/conversation_view/feed_item_row/feed_item_row.vue]: updated

**Review by iropolo — APPROVED** [review]
With the removed `!important` and the clarification on why we are updating `-lock`, Im okay with this.

Thanks!

**Review by braddialpad — APPROVED** [review]
Looks good thanks!

**Pull request merged**